tests/test_embedding_service.py

```diff
@@ -160,9 +160,9 @@ def test_query_embedding_validation(self, client):
                 response = client.post(
                     "/api/v1/embed/query", json=case["input"]
                 )
-            assert (
-                response.status_code == case["expected_status"]
-            ), f"Failed on: {case['name']}"
+            assert response.status_code == case["expected_status"], (
+                f"Failed on: {case['name']}"
+            )
```

Comment:
Ruff's formatter prefers to group the expression and split the message onto the next line. There are many instances of this same reformatting below.